steam/ext/csgo/enums.py

```diff
@@ -98,7 +98,25 @@ class ItemOrigin(IntEnum):
     LevelUpReward             = 24
     Max                       = 25
 
+class PerMapRankEnum(IntEnum):
```

Comment:
Drop the enum suffix from these 2 classes

Reply:
Done.